content/workflows/built-in-workflows.md

```diff
@@ -236,32 +238,31 @@ This sub-graph determines the operations that will be executed during the workfl
 
 Scales out/in the node subgraph of the system topology applying the `install`/`uninstall` workflows' logic respectively.
 
-If the node denoted by `node_id` is contained in a compute node (or is a compute node itself) and `scale_compute` is `true` (which is the default),
-the node graph will consist of all nodes that are contained in the compute node which contains `node_id` and the compute node itself.
-Otherwise, the subgraph will consist of all nodes that are contained in the node denoted by `node_id` and the node itself.
+If the entity denoted by `scalable_entity_name` is a node template that is contained in a compute node (or is a compute node itself) and `scale_compute` is `true`,
+the node graph will consist of all nodes that are contained in the compute node which contains `scalable_entity_name` and the compute node itself.
+Otherwise, the subgraph will consist of all nodes that are contained in the node/scaling group denoted by `scalable_entity_name`.
 
 In addition, nodes that are connected to nodes that are part of the contained subgraph will have their `establish` relationship operations executed during scale out
 and their `unlink` relationship operations executed during scale in.
 
 
 **Workflow parameters:**
 
-  - *node_id*: The ID of the node to apply the scaling logic to.
+  - *scalable_entity_name*: The name of the node/scaling group to apply the scaling logic to.
   - *delta*: The scale factor. (Default: `1`)
     - For `delta > 0`: If the current number of instances is `N`, scale out to `N + delta`.
     - For `delta < 0`: If the current number of instances is `N`, scale in to `N - |delta|`.
     - For `delta == 0`, leave things as they are.
-  - *scale_compute*: should `scale` apply on the compute compute node containing the node denoted by `node_id`. (Default: `true`)
-    - If `scale_compute` is set to `false`, the subgraph will consist of all the nodes that
-      are contained in the node denoted by `node_id` and the node denoted by `node_id` itself.
-    - Otherwise, the subgraph will consist of all nodes that are contained in the compute node that contains the node denoted by `node_id`
-      and the compute node itself.
-    - If the node denoted by `node_id` is not contained in a compute node, it is as if this parameters was set
-      to `false`.
+  - *scale_compute*: should `scale` apply on the compute node containing the node denoted by `scalable_entity_name`. (Default: `false`)
+    - If `scalable_entity_name` specifies a node, and `scale_compute` is set to `false`, the subgraph will consist of all the nodes that
+      are contained in the that node and the node itself.
+    - If `scalable_entity_name` specifies a node, and `scale_compute` is set to `true`, the subgraph will consist of all nodes that are contained in the
```

Comment:
dunno if we should address this or not.. but this entire thing is written in L241 as well..